src/components/Sidebar/Sidebar.jsx

```diff
@@ -29,14 +30,26 @@ export const Sidebar = ({
     onLogoClick,
     isStickyHeader = true,
     isStickyFooter = true,
+    shouldCloseOnChanges = true,
+    isRightDrawerVisible = null,
+    setIsRightDrawerVisible,
```

Comment:
Why do you need these two? There's too many props coming in.

Reply:
12 props :o

Reply:
in order to trigger closing of drawer from the seller app as proposed in comment above

Reply:
I was thinking on other way to fixing this via url params, maybe you have some suggestions how to optimize? in the end we need to handle case when the api call is made on the background and not close the drawer, but another case when we need to close it when purchase tile clicked while the user on the /purchases route